client/my-sites/plans-features-main/index.tsx

```diff
@@ -722,58 +729,55 @@ const PlansFeaturesMain = ( {
 								}
 							/>
 							{ ! hidePlansFeatureComparison && (
-								<ComparisonGridToggle
-									onClick={ toggleShowPlansComparisonGrid }
-									label={
-										showPlansComparisonGrid
-											? translate( 'Hide comparison' )
-											: translate( 'Compare plans' )
-									}
-									ref={ observableForOdieRef }
-								/>
-							) }
-							{ ! hidePlansFeatureComparison && showPlansComparisonGrid ? (
-								<div
-									ref={ plansComparisonGridRef }
-									className="plans-features-main__comparison-grid-container"
-								>
-									<ComparisonGrid
-										gridPlans={ gridPlansForComparisonGrid }
-										gridPlanForSpotlight={ gridPlanForSpotlight }
-										paidDomainName={ paidDomainName }
-										wpcomFreeDomainSuggestion={ wpcomFreeDomainSuggestion }
-										isCustomDomainAllowedOnFreePlan={ isCustomDomainAllowedOnFreePlan }
-										isInSignup={ isInSignup }
-										isLaunchPage={ isLaunchPage }
-										onUpgradeClick={ handleUpgradeClick }
-										flowName={ flowName }
-										selectedFeature={ selectedFeature }
-										selectedPlan={ selectedPlan }
-										siteId={ siteId }
-										isReskinned={ isReskinned }
-										intervalType={ intervalType }
-										hideUnavailableFeatures={ hideUnavailableFeatures }
-										currentSitePlanSlug={ sitePlanSlug }
-										planActionOverrides={ planActionOverrides }
-										intent={ intent }
-										showLegacyStorageFeature={ showLegacyStorageFeature }
-										showUpgradeableStorage={ showUpgradeableStorage }
-										stickyRowOffset={ masterbarHeight }
-										usePricingMetaForGridPlans={ usePricingMetaForGridPlans }
-										allFeaturesList={ FEATURES_LIST }
-										planTypeSelectorProps={ planTypeSelectorProps }
-										onStorageAddOnClick={ ( addOnSlug ) =>
-											recordTracksEvent( 'calypso_signup_storage_add_on_dropdown_option_click', {
-												add_on_slug: addOnSlug,
-											} )
-										}
-									/>
+								<>
 									<ComparisonGridToggle
 										onClick={ toggleShowPlansComparisonGrid }
-										label={ translate( 'Hide comparison' ) }
+										label={
+											showPlansComparisonGrid
+												? translate( 'Hide comparison' )
+												: translate( 'Compare plans' )
+										}
+										ref={ observableForOdieRef }
 									/>
-								</div>
-							) : null }
+									<div ref={ plansComparisonGridRef } className={ comparisonGridContainerClasses }>
```

Comment:
Non Blocking! Could we just add the `display: none` as an inline style?

Reply:
Not sure I know of a good reason to use inline styles for anything 🤔

Reply:
I was thinking from a standpoint of simplicity, for this specific instance?

For this simple task we are using the indirection of the `classnames` library and sass instead of just

`style = {{display: isHidden ? 'none' : 'block'}}`

Reply:
a styled component could have worked though. not entirely sold on those yet either being sprinkled in along with styles living separately